modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/frontend/token/definition/CXFrontendTokenCategoryImpl.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.client.extension.type.internal.frontend.token.definition;
+
+import com.liferay.client.extension.type.frontend.token.definition.FrontendToken;
+import com.liferay.client.extension.type.frontend.token.definition.FrontendTokenCategory;
+import com.liferay.client.extension.type.frontend.token.definition.FrontendTokenMapping;
+import com.liferay.client.extension.type.frontend.token.definition.FrontendTokenSet;
+import com.liferay.client.extension.type.internal.frontend.token.definition.json.JSONLocalizer;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Locale;
+
+/**
+ * @author Iván Zaera
+ */
+public class CXFrontendTokenCategoryImpl implements FrontendTokenCategory {
+
+	public CXFrontendTokenCategoryImpl(
```

Comment:
Brian asked me to change CX to ClientExtension in a JavaScript PR some days ago, so maybe you want to reconsider your naming...

Reply:
Thanks, that will save me a review from him haha